cmd/common.go

```diff
@@ -154,3 +169,51 @@ func prettyPrint(r *check.Controls, summary check.Summary) {
 		summary.Pass, summary.Fail, summary.Warn,
 	)
 }
+
+func savePgsql(jsonInfo	 string) {
```

Comment:
if you don't mind I think this is big enough to warrant starting a new file called, say, postgres.go or database.go. 

(I don't really like that this file is called common.go either, but that's another matter!)

Reply:
yep